src/main/java/beans/CommentBean.java

```diff
@@ -1,12 +1,17 @@
 package beans;
 
-public class CommentBean {
+import java.io.Serializable;
+
+import helper.BeanHelper;
+
+public class CommentBean extends BeanHelper implements Serializable {
 	
 	public static final long serialVersionUID = 1L;
 	
 	private int userId;
 	private int commentatorId;
 	private boolean vote;
+	private boolean action;
```

Comment:
naming seems a bit odd for a boolean value, what does true action or true vote actually mean, for vote it could possibly be upvote or something similar if that is the case as for action I actually cannot figure out what it means just from seeing this variable.